com.unity.netcode.adapter.utp/CHANGELOG.md

```diff
@@ -2,4 +2,4 @@
 All notable changes to this package will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [0.0.1-preview.1] - 2020-12-20
-This is the first release of Unity MLAPI Package
+This is the first release of Unity Transport for Netcode for Gameobjects
```

Comment:
nit:
```suggestion
This is the first release of Unity Transport for Netcode for GameObjects
```

also shouldn't this changelog entry be different? 1.0.0-pre and new date (today?)

Reply:
We will fix the changelog in another PR.